app/controllers/dragons_teams_controller.rb

```diff
@@ -0,0 +1,51 @@
+class DragonsTeamsController < ApplicationController
+  def index
+    @view_model = UserHomePageViewModel.new
+  end
+
+  def show
+    @view_model = UserHomePageViewModel.new
+    @user = current_user
+    @dragon = @user.dragons.find(params[:id])
+  end
+
+  def new
+    @view_model = UserHomePageViewModel.new
+    @user = @view_model.current_user
```

Comment:
`@user = @view_model.current_user`
to nie jest potrzebne `current_user` jest dostępny bezpośrednio w widoku